Utilities/AzureOnboardingCLI/azure-automatic-role-assignment.py

```diff
@@ -190,25 +228,32 @@ def build_custom_role(custom_role_name, custom_role_json_local_path, subscriptio
 
 
 def create_required_parameters_for_spot_registrations(
-    subscription,
-    custom_role_name,
-    custom_role_json_local_path,
-    service_principal_name,
-    app_registration_id=None,
+        subscription,
+        custom_role_name,
+        custom_role_json_local_path,
+        service_principal_name,
+        products,
+        app_registration_id=None
 ):
```

Comment:
What changed with the format? spaces vs tabs?

Reply:
My auto-format must have changed it.  I reverted the double indent.